configuration.nix

```diff
@@ -12,40 +12,45 @@ in
     ./nftables.nix
     # ./nat.nix
     # ./firewall.nix
-    # ./dhcp.nix
+    ./dhcp.nix
   ];
 
-  options.pi-router = {
+  options.pi-router = with lib;{
 
-    interfaces =
-      with lib;
-      {
-        lan = {
-          ip = mkOption {
-            type = types.str;
-            description = "IPv4 address of the LAN interface";
-            default = "192.168.101.1";
-          };
-          name = mkOption {
-            type = types.str;
-            description = "Interface used for LAN";
-            default = "end0";
-          };
+    dnsServers = mkOption {
+      type = types.listOf types.str;
+      description = "List of DNS servers";
+      default = [ "1.1.1.1" "1.0.0.1" ];
+      example = [ "9.9.9.9" "1.0.0.1" ];
+    };
```

Comment:
We should discuss this option in combination with the Unbound server.
Our router provides a DNS server our clients should use.
I still live the idea of a DNS option, but we should implement a different behavior.

Reply:
removed for now.